# This message is used to configure the physics Engine
float64 timestep #Time step for physics engine
float64 realtimerate
